README.md

```diff
@@ -36,11 +36,49 @@ $ make run-eth
 ```
 
 ### Build and Run an example
+install rush if not already
+```
+npm install -g @microsoft/rush
+```
+
+install all dependencies
+```
+rush update
+```
 
+compile and build all contracts
+```
+rushx build    # only rebuild if source file changed
+rushx rebuild  # force rebuild all
+```
+
+run all tests
+```
+rushx test
+```
+
+build and run together
+```
+rushx run
+```
+
+or we can run tests for a single example:
 - cd into one of the example project
-- Install dependencies with `yarn`
-- Compile contract with `yarn build`
   - You can find your contract ABI in the build directory. You can upload these ABI files to [acala evm playground](https://evm.acala.network/#/upload) for testing.
-- Run the tests with `yarn test` or `yarn test --with-ethereum-compatibility` with the chain which enable evm compatibility mode.
+  - Run the tests with `rushx test` or `rushx test --with-ethereum-compatibility` with the chain which enable evm compatibility mode.
```

Comment:
We could remove the evm compatibility flags throughout the README, since that's what we are striving for (and it's working already). 🤔 

```suggestion
  - Run the tests with `rushx test`.
```